wiki/webserver/ExternalWebserversSQL.md

```diff
@@ -101,3 +101,45 @@ ProxyPassMatch ^/(maps/[^/]*/live/.*) http://127.0.0.1:8100/$1
 > **Important:**<br>
 > The above config is **just an example** and not a complete config you can just copy&paste. You **will** need to adapt it to your setup!
 {: .info .important }
+
+## Caddy with php-fpm
+
+Here is an example for how it could look like on Caddy with php-fpm:
+```caddy
+# Replace this with your own domain.
+# Put http://example.com if you are behind a reverse proxy, that way Caddy will not try to process HTTPS
+example.com {
+    root * /home/container/public
+    # https://caddyserver.com/docs/caddyfile/patterns#php-fpm
+    php_fastcgi unix//run/php/php7.4-fpm.sock # May need to be changed
+    file_server
+    # Rewrite requests to sql.php
+    try_files {path} /sql.php
+}
+```
+> **Important:**<br>
+> The above config is **just an example** and not a complete config you can just copy&paste. You **will** need to adapt it to your setup!
+{: .info .important }
+
+## FrankenPHP
+
+Here is an example for how it could look like on FrankenPHP:
+```caddy
+{
+    # https://caddyserver.com/docs/caddyfile/patterns#frankenphp
+    frankenphp
+    order php_server before file_server
+}
+# Replace this with your own domain.
+# Put http://example.com if you are behind a reverse proxy, that way Caddy will not try to process HTTPS
+example.com {
+    root * /home/container/public
+    file_server
+    php_server
+    # Rewrite requests to sql.php
+    try_files {path} /sql.php
+}
```

Comment:
```suggestion
example.com {
	# This designates the root for the webserver
	root /var/www/map.hasphetica.win
	file_server

	# https://caddyserver.com/docs/caddyfile/patterns#php-fpm
	# You may need to modify this path. Here are some examples;
	# /unix//run/php-fpm/php-fpm.sock
	# /unix//run/php/php8.2-fpm.sock
	php_fastcgi unix//run/php/php7.4-fpm.sock


	# NOT OPTIONAL:
	# This directive tells the webserver to use SQL.PHP, which is the main file that handles PHP requests. You need this for PHP.
	handle {
		try_files {path} /sql.php
	}

	# OPTIONAL:
	# This handles logging. Modify INFO to ERROR if you only want error logs, or remove this if you do not want logging.
	log {
		output file /var/log/caddy/map-access.log
		format console
		level INFO
	}

	# OPTIONAL:
	# Proxy requests for live data to the bluemaps integrated webserver.
	# If you have multiple servers you will need to proxy each map-id to the correct server.
	handle /maps/*/live/* {
		reverse_proxy 127.0.0.1:8100
	}
}
```

This should be modified to match my review for caddy. I do not use FrankenPHP, So I cannot test this- however, in theory, it should work. @ILikePlayingGames please test my proposed config above before signing off on this.